rpm/python-safeeyes.spec

```diff
@@ -0,0 +1,100 @@
+Name:           python-safeeyes
+Version:        3.2.0
+Release:        %autorelease
+Summary:        Take periodic breaks to protect your eyes
+
+License:        GPL-3.0-or-later
+URL:            https://github.com/slgobinath/SafeEyes
+Source:         %{pypi_source safeeyes}
+
+BuildArch:      noarch
+
+BuildRequires:  cairo-gobject-devel
+BuildRequires:  gobject-introspection-devel
+BuildRequires:  gtk4
+BuildRequires:  python3-devel
+BuildRequires:  desktop-file-utils
+BuildRequires:  gettext
+
+# for notification plugin
+BuildRequires: libnotify
+Requires: libnotify
+
+# for audiblealert plugin
+Requires: (ffmpeg-free or pipewire-utils)
+
+# for smartpause plugin (optional in pyproject.toml)
+BuildRequires: python3-pywayland
+Requires: python3-pywayland
+
+# xprintidle does not exist on fedora
+#Suggests: xprintidle
+
+# for healthstats plugin (optional in pyproject.toml)
+BuildRequires: python3-croniter
```

Comment:
All of these extra requires appear to do nothing at runtime.
Running `rpm --query --package rpm/noarch/python3-safeeyes-3.2.0-1.fc44.noarch.rpm --requires` gives:

```
/usr/bin/python3
python(abi) = 3.13
python3.13dist(babel)
python3.13dist(packaging)
python3.13dist(pygobject)
python3.13dist(python-xlib)
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PartialHardlinkSets) <= 4.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsZstd) <= 5.4.18-1
```
These are only the dependencies from `%generate_buildrequires`, but none of the manually added ones here.